config-controller/api/v1alpha1/policy_types.go

```diff
@@ -81,6 +92,10 @@ func (p SecurityPolicySpec) ToProtobuf() *storage.Policy {
 			Name: exclusion.Name,
 		}
 
+		if protoTS, err := protocompat.ParseRFC3339NanoTimestamp(exclusion.Expiration); err == nil {
```

Comment:
FWIW I tried to parse a date I found in a k8s resource using `time.Parse(time.RFC3339Nano, t)` and it seemed to be compatible.